photon-client/src/integTest/scala/com/linkedin/photon/ml/DriverTest.scala

```diff
@@ -29,7 +29,7 @@ import com.linkedin.photon.ml.PhotonOptionNames._
 import com.linkedin.photon.ml.TaskType.TaskType
 import com.linkedin.photon.ml.constants.MathConst
 import com.linkedin.photon.ml.diagnostics.DiagnosticMode
-import com.linkedin.photon.ml.io.{FieldNamesType, GLMSuite, InputFormatType}
+import com.linkedin.photon.ml.photon_io.{FieldNamesType, InputFormatType}
```

Comment:
Underscores shouldn't be used in package names. Also, why are we distinguishing IO from Photon IO? The name "Photon" encompasses everything in the project, including GAME.